documentation/src/main/asciidoc/entity-view/manual/en_US/05_fetch_strategies.adoc

```diff
@@ -4,7 +4,7 @@
 There are multiple different fetch strategies available for fetching. Right now, a fetch strategy can only be applied on a correlated mapping,
 but soon will also be able to be specified for any other mapping. Also see https://github.com/Blazebit/blaze-persistence/issues/323[#323] for further information.
 
-NOTE: `@MappingSubquery` is the only mapping that doesn't support overriding the fetch strategy. It will always use the `JOIN` strategy
+NOTE: `@MappingSubquery` is the only mapping that doesn't support overriding the fetch strategy. It will always use the `JOIN` strategy i.e. will put the subquery into the main query.
```

Comment:
Change to:
"It will always use the `JOIN` strategy, i.e. it will put the subquery into the main query."